tests/common/utils/test_excel.py

```
@@ -0,0 +1,37 @@
+    """Test suite for Excel utility functions."""
+
+    def test_create_workbook(self):
+        """Test that create_workbook returns a valid workbook object."""
+        wb = create_workbook()
+        assert wb is not None
+        assert hasattr(wb, "active")
+
+    def test_save_workbook(self):
+        """Test that save_workbook correctly saves a workbook to a file."""
+        with tempfile.TemporaryDirectory() as temp_dir:
+            with mock.patch("django.conf.settings.MEDIA_ROOT", temp_dir):
```

Ruff (0.8.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)